sw/device/lib/crypto/BUILD

```diff
@@ -4,15 +4,12 @@
 
 package(default_visibility = ["//visibility:public"])
 
-load("//rules:opentitan.bzl", "OPENTITAN_CPU", "OPENTITAN_PLATFORM", "opentitan_binary")
+load("//rules/opentitan:defs.bzl", "OPENTITAN_CPU", "OPENTITAN_PLATFORM")
+load("//rules/opentitan:static_library.bzl", "ot_static_library")
 
 # Top-level cryptolib target.
-cc_library(
+ot_static_library(
     name = "otcrypto",
-    hdrs = [
-        "otcrypto.h",
```

Comment:
Can we now delete this header file, or is it still useful? The purpose was to give callers the ability to import the entire cryptolib with one import rather than revealing the internal structure.

Reply:
We can keep it if you feel it better serves the customer.

As discussed offline, we'll have to determine what a release of cryptolib looks like to downstream customers and possibly consider packaging the library archive and headers as a release artifact.  We can tackle that work when we understand how a downstream customer will want to incorporate cryptolib into their project.

Reply:
Output is named `./bazel-bin/sw/device/lib/crypto/otcrypto.a`  instead of `libotcrypto.a`